Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -62,6 +62,26 @@ public IssuesClientTests()
     }
 
     [IntegrationTest]
+    public async Task CanLockAndUnlockIssue()
+    {
+         var newIssue = new NewIssue("a test issue") { Body = "A new unassigned issue" };
+         var issue = await _issuesClient.Create(_context.RepositoryOwner, _context.RepositoryName, newIssue);
+         var retrieved = await _issuesClient.Get(_context.RepositoryOwner, _context.RepositoryName, issue.Number);
+ 
+         Assert.Equal(false, issue.Locked);
```

Comment:
You can use `Assert.False` or `Assert.True` here to make these a bit more succinct.